### DISCUSSION TITLE
"Needs work" needs work.

### DISCUSSION THREAD

**Discussion by comradekingu** [WeblateOrg/weblate#11035] [Ideas]
(I got it wrong when I ceded the argument that there are more important issues, because there is always something broken
about implementation that can be helped. Long term it fails.)

_No shared indication of trust is a manual downfall of the commons in the auto-generated age_.

How much is a translator worth?
Next to nothing, and I say this with some insight.
Most are worth _less_.

The distinction from good enough to better _is_ worth something, meaning some translators are somehow worth a lot.

As a future further stratification, when quality ultimately matters, this distinction of being better than the fold is all that matters.
(Paying anyone for less than what can be had for nothing needs no sell.)
If you can't entertain drawing in the very best, "translators" is not what you want.
Hence why what everyone thinks translators do is misnomer.
You want people that care about language looking at improving language, this is not just moving from A-B.
It is caring what A is for the capacities of language in all its beauty, and/or the comprehension of an audience.
When one individual to stay on top of changes, from any of those angles, any current platform is not a good place to learn.
Translators actually currently threaten the very idea of manual translation platforms by contributing.

Each to their own, the select few that can even do well, bear the burden of ensuring quality across projects,
across and between entire languages.
Lose a single one of those, and it falls apart.
At stake is how natural language works in terms of culture, since this data is used to create something that lives on by itself.

Does any project have even one of those for each language. No.
This reliance on the individual is ultimately shoved onto
something, a language coordinator, a developer, usually just one guy.
It is actually most often the idea of having someone there. Again.
Who knows whether you want to spend a lot of time finding perfection.
I do, and you don't. Let the people that care appear out of nowhere and help you do
something you have no capacity to do even in your language.
Sounds good.

Nobody should try to be perfect before or after.
That distinction means the workload is too great.
It also means ensured failure.
Right now for any trade-off arrived at, it usually means no great linguist translating worse yet source strings,
and _nobody_ checking up on it before merging.
Just full send ship and repeat.
Nothing standing between the distinction of good enough and better yet, means
nobody really knows whether true malice is the distinction I am talking about,
because we often check for neither.

There is no compelling reason anyone would want to be good at translation,
and there is dwindling reason to even be involved.
The tool relies on single individuals, and it does not a whole lot to be a mutual
and reciprocal system of self-improvement.

To deal with malice, someone needs to know who looked at what.
Anything not ensured at the individual level to be good is potential for flaw.

Every platform makes it easy for malice and mistakes to hide.
Get it wrong the first time, miss all e-mails about it, and the error is
now suddenly as public-facing as could be.

Everyone needs to know what everyone looked at, and this level of trust is precisely
what can tell you something about where to look for errors, and how it would help others.
At some level I have trouble remembering what I even looked at myself.

Do you not aspire to greatness and have the time to do it, but want to do something? Good.
Vouch for work that has already been done.
Otherwise stay away. Unfortunately.
We can all hold hands and pretend more people helping pulls in the same direction,
and that is only true in terms of making it all worse than irrelevant.

Completion is not quality.
Coverage says nothing about what it consists of.
100% could mean someone guessed between most easy strings requiring less work, and the rest.
Age is only an indicator of how good the system is at finding flaws over time, _and that system isn't good_.
As overall quality generally goes up, the potential for inconsistency grows with more contributors.
You can't rely on yourself not do do the thing that as organically happens between a well-meaning group.
We need a better system.

All the groundwork of checking every single string, is already being done.
Just nobody knows about it, and you hope it is all translators in every language.
It isn't.  We need the numbers to know what we are looking at.
I know some trash when I see it, and I can tell what tool produced the result I am looking at.
I'd rather be looking at how to fix it, instead of working against everyone.

We have one checkbox separating "good to go" from "hold on a minute",
and anyone can click that box. Absurd.

**Comment by nijel** [comment]
Perhaps this was meant as a followup to https://github.com/WeblateOrg/weblate/issues/2699?
